nbcparser/pkg/parser/templates/cse_cmd.sh.gtpl

```diff
@@ -148,7 +148,7 @@ IS_KATA="{{.GetIsKata}}"
 ARTIFACT_STREAMING_ENABLED="{{.GetEnableArtifactStreaming}}"
 SYSCTL_CONTENT="{{getSysctlContent .CustomLinuxOsConfig.GetSysctlConfig}}"
 PRIVATE_EGRESS_PROXY_ADDRESS="{{.GetPrivateEgressProxyAddress}}"
-BOOTSTRAP_PROFILE_CONTAINER_REGISTRY_SERVER=""
-ENABLE_IMDS_RESTRICTION="false"
-INSERT_IMDS_RESTRICTION_RULE_TO_MANGLE_TABLE="false"
+BOOTSTRAP_PROFILE_CONTAINER_REGISTRY_SERVER="{{.IMDSRestrictionConfig.GetBootstrapProfileContainerRegistryServer}}"
+ENABLE_IMDS_RESTRICTION={{.IMDSRestrictionConfig.GetEnableImdsRestriction}}
+INSERT_IMDS_RESTRICTION_RULE_TO_MANGLE_TABLE={{.GetInsertImdsRestrictionRuleToMangleTable}}
```

Comment:
I think in the future we should execute /opt/azure/containers/provision_start.sh directly from the program.

And set ENV variables in `cmd.ENV` instead of using go templates.

Reply:
How to set the variables in `cmd.ENV`?

Reply:
something like this:

```go
	cmd.Env = []string{
		"BOOTSTRAP_PROFILE_CONTAINER_REGISTRY_SERVER=" + nbc.IMDSRestrictionConfig.GetEnableImdsRestriction,
	}
```

It's an idea for the future